### PR TITLE
Customizable http router name

### PR DESCRIPTION
**Pull request by nicolaracco**
It would be helpful to allow the customization of the http router without needing to override the entire config

### CODE REVIEW

File: charts/apisix/values.yaml [hunk 1]
```diff
@@ -56,6 +56,8 @@ apisix:
         - key: ""
           path: ""
 
+  httpRouter: radixtree_uri
```
Comment (tao12345666333) [inline charts/apisix/values.yaml:59]: I think it would be better to add comments to here. 

list all options

**Review by tao12345666333 — COMMENTED** [review]
Thanks 

Please update chart's README as well

**Comment by nicolaracco** [comment]
Hello @tao12345666333. I've updated both the values.yml and the README

**Comment by tao12345666333** [comment]
I mean this one https://github.com/apache/apisix-helm-chart/blob/master/charts/apisix/README.md

**Comment by nicolaracco** [comment]
Sorry :) I've updated the right file this time

**Review by tao12345666333 — APPROVED** [review]
LGTM

**Review by nic-6443 — APPROVED** [review]

**Pull request merged**